### ISSUE TITLE
Add live preview to microsites 

### ISSUE THREAD

**Issue by willguv** [localgovdrupal/localgov_microsites#455]
@markconroy created a great prototype showing how some design elements could be edited in a live view

https://www.youtube.com/watch?v=d5TKkUXPIiM

Many Wordpress themes offer this kind of customisation experience - a panel opens to the left of the page you're viewing and changes update automatically

<img width="347" alt="Screenshot 2024-06-03 at 14 37 47" src="https://github.com/localgovdrupal/localgov_microsites/assets/708140/eb0beb0f-347a-4c0d-9743-eb58f9f0d214">

<img width="347" alt="Screenshot 2024-06-03 at 14 37 28" src="https://github.com/localgovdrupal/localgov_microsites/assets/708140/6af20edb-5087-45bc-8817-300a3001c7e2">

<img width="345" alt="Screenshot 2024-06-03 at 14 38 01" src="https://github.com/localgovdrupal/localgov_microsites/assets/708140/7dac07d1-e661-4203-8d7d-a069e813a02b">

It would be good to explore/ size this work

**Comment by willguv** [comment]
@finnlewis to rewrite

**Comment by finnlewis** [comment]
Since the original post, we've released https://github.com/localgovdrupal/localgov_live_preview

There's now a pull request here to enable the correct permissions: 

https://github.com/localgovdrupal/localgov_live_preview/pull/14

And another pull request here to bring in the module and enable it on a default install: 

https://github.com/localgovdrupal/localgov_microsites/pull/505

**Comment by finnlewis** [comment]
This is still in progress - more eyes on it from https://github.com/localgovdrupal/localgov_microsites/pull/505

**Comment by willguv** [comment]
@msayoung please have a look at this next - thanks

**Comment by msayoung** [comment]
It's been added, but not released due failing tests. 
"Something needs updating to include dependencies" says ekes.

**Comment by finnlewis** [comment]
See https://github.com/localgovdrupal/localgov_microsites/actions/runs/15706300207/job/44252918072?pr=512#step:5:17 

@tonypaulbarker @stephen-cox any ideas what config dependencies might be missing?

**Comment by tonypaulbarker** [comment]
Line 17 here https://github.com/localgovdrupal/localgov_microsites/actions/runs/15706300207/job/44252918072?pr=512#step:5:17 is complaining about microsites live preview dependencies and lists the config dependencies. 
There are a lot of config dependencies that are coming from modules not listed as dependencies in the microsites live preview .info.yml  
https://github.com/localgovdrupal/localgov_live_preview/blob/1.x/modules/localgov_live_preview_microsites/config/install/core.entity_form_display.group.microsite.localgov_live_preview.yml 
We could also update the order of dependencies in LocalGov Drupal Microsites .info.yml so that   
- localgov_live_preview_microsites:localgov_live_preview_microsites
is after the things that it depends on.

Will try updating these and see if we get a pass.
